client/src/components/MovieTimer/MovieTimer.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useContext, useEffect } from 'react';
+import './MovieTimer.css';
+import TimerMachine from 'react-timer-machine';
+import moment from "moment";
+import momentDurationFormatSetup from "moment-duration-format";
+import { PlayContext } from '../../containers/PlayMovie/PlayMovie';
+momentDurationFormatSetup(moment);
+
+const MovieTimer = ({ runtime , movie }) => {
+
+  const { current, setCurrent, checkComments, resetComments } = useContext(PlayContext);
+  const [started , setStarted] = useState(false);
+  const [paused , setPaused] = useState(false);
+  const [timeStart, setTimeStart] = useState(0);
+  const timeEnd = 60000 * runtime;
+
+  const startPlayer = () => {
+    setStarted(!started);
+  };
+
+  const pausePlayer = () => {
+    setPaused(!paused);
+  };
+
+  const handlePosition = (now, end) => {
+    const position = now / end;
+    const fullWidth = document.querySelector('.progress-container');
+    const progress = document.querySelector('.my-progress');
+    progress.style.width = Math.floor(position * fullWidth.offsetWidth) + 'px';
+  };
+
+  const handlePositionComplete = () => {
+    document.querySelector('.my-progress').style.width = '100%';
+  };
+  const handlePositionReset = () => {
+    document.querySelector('.my-progress').style.width = '0%';
+  };
+
+  const handleControl = (e) => {
+/*    const fullWidth = document.querySelector('.progress-container');
+    const progress = document.querySelector('.my-progress');
+    console.log(Math.floor(60000 * runtime * e.clientX / fullWidth.offsetWidth))
+    progress.style.width = e.clientX + 'px';
+    setCurrent(() => {
+      setTimeStart(Math.floor(60000 * runtime * e.clientX / fullWidth.offsetWidth))
+      return Math.floor(60000 * runtime * e.clientX / fullWidth.offsetWidth);
+    });*/
+  };
+
+  return (
+    <div className="movie-timer-container">
+      <div className="movie-infos">
+      {movie.title}
+      </div>
+      <TimerMachine
```

Comment:
It would be great exercise to redo this thing by yourself.

Reply:
Also, you would take advantage of a simpler API, this one is a bit of an overkill.